ignite/metrics/gan/fid.py

```diff
@@ -0,0 +1,213 @@
+from typing import Callable, Optional, Sequence, Union
+
+import numpy as np
+import torch
+
+from ignite.metrics.metric import Metric, reinit__is_reduced, sync_all_reduce
+
+__all__ = ["FID", "InceptionExtractor"]
+
+
+def fid_score(
+    mu1: torch.Tensor, mu2: torch.Tensor, sigma1: torch.Tensor, sigma2: torch.Tensor, eps: float = 1e-6
+) -> float:
+
+    try:
+        import scipy
```

Comment:
@gucifer This line is not covered by the tests.

Please have a look to the following test for testing missing modules
https://github.com/pytorch/ignite/blob/86cec1165441a945c93c938c23544ac52a17a170/tests/ignite/contrib/metrics/test_cohen_kappa.py#L23